jboss-eap-clustered-multivm-rhel8/README.md

```diff
@@ -70,7 +70,7 @@ Click this [Red Hat KB article on RHSM](https://access.redhat.com/products/red-h
 
 ## Prerequisites
 
-1. Azure Subscription with the specified payment method (RHEL 8 is an [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/RedHat.RedHatEnterpriseLinux80-ARM?tab=Overview) product and requires a payment method to be specified in the Azure Subscription). If you select the RHEL OS License type as BYOS (Bring-Your-Own-Subscription), please follow steps mentioned under section 'Subscriptions and Costs'.
+1. Azure Subscription with the specified payment method.  RHEL 8 is an [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/RedHat.RedHatEnterpriseLinux80-ARM?tab=Overview) product and requires a payment method to be specified in the Azure Subscription. If you select the RHEL OS License type as BYOS (Bring-Your-Own-Subscription), please follow the steps mentioned under section 'Subscriptions and Costs'.
```

Comment:
We have this line completely changed in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback so that we can replicate the same format here.